test/integration/singlecluster/controller/jobs/job/job_controller_test.go

```diff
@@ -1394,8 +1394,6 @@ var _ = ginkgo.Describe("Interacting with scheduler", ginkgo.Ordered, ginkgo.Con
 			ginkgo.By("checking that the second workload is admitted", func() {
 				gomega.Eventually(func(g gomega.Gomega) {
 					g.Expect(k8sClient.Get(ctx, wlLookupKey2, createdWorkload2)).Should(gomega.Succeed())
-					g.Expect(createdWorkload2.Spec.PriorityClassName).Should(gomega.Equal(highWorkloadPriorityClass.Name))
```

Comment:
Instead of dropping the assert on `.Spec.PriorityClassName` can we replace to assert on the PriorityClassRef?

Reply:
We already have it https://github.com/epam/kubernetes-kueue/blob/a5230b0f52da33eef56d14b6bfa88cdee040517c/test/integration/singlecluster/controller/jobs/job/job_controller_test.go#L1390-L1392.